examples/with-msw/pages/index.js

```diff
@@ -32,12 +42,22 @@ export default function Home({ book }) {
 
 export async function getServerSideProps() {
   // Server-side requests are mocked by `mocks/server.js`.
-  const res = await fetch('https://my.backend/book')
-  const book = await res.json()
+  // This request will fail in production, as MSW is only run in development.
+  // In a real-world app this request would hit the actual backend.
+  try {
+    const res = await fetch('https://my.backend/book')
+    const book = await res.json()
 
-  return {
-    props: {
-      book,
-    },
+    return {
+      props: {
+        book,
+      },
+    }
+  } catch (error) {
+    return {
+      props: {
+        error: true,
+      },
```

Comment:
Let's return something more friendly, like

```
props: {
  inProduction: true,
},
```

We don't have an error here, it's expected to fail because there's no real endpoint